astronomer/providers/google/cloud/sensors/gcs.py

```diff
@@ -123,3 +124,39 @@ def execute_complete(
         if event["status"] == "success":
             return event["matches"]
         raise AirflowException(event["message"])
+
+
+class GCSObjectUpdateSensorAsync(GCSObjectUpdateSensor):
```

Comment:
Comment and doc-string is missing on `GCSObjectUpdateSensorAsync`.

Reply:
I have inherited the syn version I have added only the async version

Reply:
We should still add it

Reply:
Fixed now